templates/web/base/front/emergency-message.html

```diff
@@ -0,0 +1,6 @@
+[% emergency_message = c.cobrand.emergency_message %]
+[% IF emergency_message %]
+<div class="emergency-message">
+  [% emergency_message | safe %]
```

Comment:
Since we allow admins to enter line breaks in their message, do you think we should convert double linebreaks into paragraphs here? Right now, the linebreaks get output into the HTML, and then collapsed into a single whitespace character by the browser.

You _could_ get the browser to render the linebreaks using `white-space: pre-line;`, but then you’ll need to remove any whitespace immediately before/after the message inside the `<div>`, and it’s also a bit nasty to have a full line height gap between the "paragraphs", rather than whatever the normal paragraph `margin-bottom` would be.

Speaking of which, if you wrap paragraphs in `<p>` tags, you’ll probably want to add a style that removes the margin-bottom from `.emergency-child > :last-child`.